src/lib/context.ts

```diff
@@ -214,7 +214,8 @@ export class AppContext {
 
     getMetadata() {
         if (this.span) {
-            const metadata = {};
+            const requestId = this.get('requestId');
+            const metadata = {requestId};
```

Comment:
Maybe we should check here if requestId is present in the context?

```suggestion
            const metadata = requestId ? {requestId} : {};
```

Reply:
This may only be important for an empty string, but it makes sense